src/ai/backend/storage/api/client.py

```diff
@@ -346,6 +346,8 @@ async def prepare_tus_session_headers(
     headers["Upload-Length"] = str(token_data["size"])
     return headers
 
+async def health_check(request: web.Request) -> web.Response:
```

Comment:
This is totally my preference, but I think it would be enough to name the handler as `status`, instead of `health_check`, since it is shorter...